docs/reference/02-cli.md

```
@@ -179,6 +173,24 @@ Expected output contains health lines for:
 - `Prefect server`
 - `RustFS (S3)`
 
+## `ccc system worker start`
+
+Starts a course-scoped Prefect worker on the work pool named in that course's
+configuration. For `shared_environment` courses it mirrors any `RUSTFS_*`
+credentials in the environment into the `AWS_*` names boto3 reads; if none are
+set it changes nothing, leaving IAM roles and `~/.aws` profiles intact.
+
+```bash
+$ ccc system worker start --course ccc-course-cs101
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)